oxidation/libparsec/crates/client_types/src/local_device.rs

```diff
@@ -1,12 +1,15 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
 
 use serde::{Deserialize, Serialize};
-use serde_with::*;
+use serde_with::serde_as;
 use serialization_format::parsec_data;
 use sha2::Digest;
 
-use libparsec_crypto::*;
-use libparsec_types::*;
+use libparsec_crypto::{PrivateKey, PublicKey, SecretKey, SigningKey, VerifyKey};
```

Comment:
didn't we say we would use `use libparsec_crypto::prelude::*` instead ?

Reply:
Yes but `libparsec_crypto` doesn't provide a module `prelude`

Reply:
And it was simpler to correct with rust-analyzer `expand glob import`